pkgs/development/node-packages/node-env.nix

```diff
@@ -363,7 +363,7 @@ let
 
         npm ${forceOfflineFlag} --nodedir=${nodeSources} ${npmFlags} ${stdenv.lib.optionalString production "--production"} rebuild
 
-        if [ "''${dontNpmInstall-}" != "1" ]
+        if [ "$dontNpmInstall" != "1" ]
```

Comment:
This will produce an error if `$dontNpmInstall` is unset and `set -u` is enabled (which I think it is by default for stdenv packages, not sure about node packages).